packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount.jsx

```diff
@@ -182,22 +187,7 @@ const Amount = ({
                     />
                 </React.Fragment>
             )}
-            {contract_type === 'vanilla' && (
```

Comment:
MinMaxStakeInfo generalized component will be used for both Turbos & Vanillas, as discussed with designers.